uperf-wrapper/uperf-wrapper.py

```diff
@@ -206,13 +208,20 @@ def main():
         server_node = os.environ["server_node"]
     if "client_node" in os.environ:
         client_node = os.environ["client_node"]
+    if "bo_ip" in os.environ:
+        redis_host = os.environ["bo_ip"]
+
+    redis_port = 6379  
```

Comment:
make this redis part of code optional, so that if folks are running outside of ripsaw, they should still be able to run with this.